elki-core-math/src/main/java/elki/math/statistics/dependence/mcde/MWPTest.java

```diff
@@ -50,19 +48,27 @@
     booktitle = "Proc. Scientific and Statistical Database Management (SSDBM 2019)", //
     url = "https://doi.org/10.1145/3335783.3335795", //
     bibkey = "DBLP:conf/ssdbm/FoucheB19")
+
 public class MWPTest implements MCDETest<MWPTest.MWPRanking> {
   /**
    * Static Constructor - use this one
    */
   public static final MWPTest STATIC = new MWPTest();
 
+  /**
+   * Share of instances in marginal restriction (reference dimension).
+   * Note that in the original paper alpha = beta and as such there is no
+   * explicit distinction between the parameters.
+   */
+  protected double beta;
```

Comment:
Another merge error? beta is not used here.
Maybe you can make a clean rebase on master, and review the differences? Also the empty line before the class name above; and code formatting differences that do not use the ELKI code formatter style.